src/components/SpiceList.js

```diff
@@ -8,19 +8,36 @@ class SpiceList extends React.Component {
     search: ""
   }
 
+  getSearchTerm = (searchTerm) => {
+    this.setState({
+      search: searchTerm
+    })
+  }
+
+  toggleFilter = (toggleStatus) => {
+    this.setState({
+      fourStarOnly: toggleStatus
+    })
+  }
```

Comment:
One trick here you could try for shortening the syntax would be to use [shorthand property names](https://ui.dev/shorthand-properties/#:~:text=With%20Shorthand%20Properties%2C%20whenever%20you,can%20omit%20the%20property%20name.) like this:

```js
getSearchTerm = search => {
  this.setState({ search })
}

toggleFilter = fourStarOnly => {
  this.setState({ fourStarOnly })
}
```

This works as long as the key in state matches the name of the variable you're trying to assign to it.